doc/api/cli.md

```diff
@@ -94,6 +94,25 @@ Follows `require()`'s module resolution
 rules. `module` may be either a path to a file, or a node module name.
 
 
+### `--inspect[=host:port]`
+<!-- YAML
+added: v6.5.0
+-->
+
+Activate inspector on host:port. Default is 127.0.0.1:9229.
+
+V8 Inspector integration allows attaching Chrome DevTools and IDEs to Node.js
+instances for debugging and profiling. It uses the [Chrome Debugging Protocol][].
+
+
+### `--inspect-brk[=host:port]`
+<!-- YAML
+added: v6.5.0
```

Comment:
I think this was introduced in [7.6.0](https://nodejs.org/en/blog/release/v7.6.0/).